src/zenml/models/v2/core/pipeline_run.py

```diff
@@ -115,6 +115,11 @@ class PipelineRunRequest(WorkspaceScopedRequest):
         default=None,
         title="Tags of the pipeline run.",
     )
+    configured_model_version_id: Optional[UUID] = Field(
```

Comment:
Are there any other ways that a model version can be linked to a pipeline run other than being explicitly configured? Do we count it for example if someone tries to load an artifact from the model version inside the step?

Reply:
As of now - we don't. Only what is in Decorator counts